fail2ban/readme.md

```diff
@@ -0,0 +1,122 @@
+## Fail2Ban Monitoring Template with Telegraf
+
+### Dashboards
+
+#### Fail2Ban
+
+   <img src="img/fail2ban-dashboard.png" width="425"/> 
+
+#### InfluxDB UI
+
+In the InfluxDB UI, go to Settings->Templates and enter this URL: https://raw.githubusercontent.com/influxdata/community-templates/master/fail2ban/img/fail2ban-template.yml
+
+#### Influx CLI
+If you have your InfluxDB credentials configured in the CLI, you can install this template with:
+
+`influx apply -f https://raw.githubusercontent.com/influxdata/community-templates/master/fail2ban/fail2ban-template.yml`
+
+## Requirements
```

Comment:
Can you move this section into the `## Setup Instructions` section? That way they will be shown in the UI when somebody is installing this template.

Reply:
Readme.md was updated.